src/main/java/ru/sbtqa/tag/pagefactory/support/SelenoidCapabilitiesProvider.java

```diff
@@ -3,85 +3,110 @@
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.drivers.TagWebDriver;
 import ru.sbtqa.tag.qautils.properties.Props;
 
+import java.util.HashMap;
+
+import static org.openqa.selenium.remote.BrowserType.OPERA;
+
 public class SelenoidCapabilitiesProvider {
 
     private static final Logger LOG = LoggerFactory.getLogger(SelenoidCapabilitiesProvider.class);
 
-    private static final String REMOTE_WEBDRIVER_BROWSER_VERSION = Props.get("webdriver.remote.browserVersion");
-    private static final String REMOTE_WEBDRIVER_ENABLE_VNC = Props.get("webdriver.remote.enableVNC");
-    private static final String REMOTE_WEBDRIVER_ENABLE_VIDEO = Props.get("webdriver.remote.enableVideo");
-    private static final String REMOTE_WEBDRIVER_SCREEN_RESOLUTION = Props.get("webdriver.remote.screenResolution");
-    private static final String REMOTE_WEBDRIVER_VIDEO_SCREEN_SIZE = Props.get("webdriver.remote.video.screenSize");
-    private static final String REMOTE_WEBDRIVER_VIDEO_FRAME_RATE = Props.get("webdriver.remote.video.screenSize");
-    private static final String REMOTE_WEBDRIVER_NAME_OF_TESTS = Props.get("webdriver.remote.nameOfTests");
-    private static final String REMOTE_WEBDRIVER_TIME_ZONE = Props.get("webdriver.remote.timeZone");
-    private static final String REMOTE_WEBDDRIVER_HOST_ENTRIES = Props.get("webdriver.remote.hostEntries");
+    private static final String SELENOID_BROWSER_VERSION = Props.get("selenoid.browserVersion");
+    private static final String SELENOID_ENABLE_VNC = Props.get("selenoid.enableVNC");
+    private static final String SELENOID_SCREEN_RESOLUTION = Props.get("selenoid.screenResolution");
+    private static final String SELENOID_ENABLE_VIDEO = Props.get("selenoid.enableVideo");
+    private static final String SELENOID_VIDEO_NAME = Props.get("selenoid.video.name");
+    private static final String SELENOID_VIDEO_SCREEN_SIZE = Props.get("selenoid.video.screenSize");
+    private static final String SELENOID_VIDEO_FRAME_RATE = Props.get("selenoid.video.frameRate");
+    private static final String SELENOID_NAME_OF_TESTS = Props.get("selenoid.nameOfTests");
+    private static final String SELENOID_TIME_ZONE = Props.get("selenoid.timeZone");
+    private static final String SELENOID_HOST_ENTRIES = Props.get("selenoid.hostEntries");
+    private static final String SELENOID_APPLICATION_CONTAINERS = Props.get("selenoid.applicationContainers");
+    private static final String SELENOID_CONTAINER_LABLES = Props.get("selenoid.containerLables");
```

Comment:
Also add this options to https://github.com/sbtqa/page-factory/blob/master/src/main/resources/config/application.properties-sample with description, or link to selenoid doc

Reply:
Done